src/cypress-runner.js

```diff
@@ -66,6 +66,12 @@ const getCypressOpts = function (runCfg, suiteName) {
     }
   };
 
+  if (runCfg.cypress.record !== undefined && runCfg.cypress.record && runCfg.cypress.key !== undefined) {
```

Comment:
Wouldn't this theoretically be enough? Or is it one of those nasty javascript pitfalls?

```suggestion
  if (runCfg.cypress.record && runCfg.cypress.key !== undefined) {
```

Reply:
It's enough just to check whether is it true or false. I fixed it.